1-weekly-mission-week2-main/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
```

Comment:
웹 접근성을 위해 lang 속성을 추가해주면 좋을 것 같아요
```suggestion
<html lang='ko'>
```